_layouts/english.html

```diff
@@ -7,16 +7,24 @@
   <div class="container">
     <div class="row justify-content-end p-0 py-md-3 py-md-8">
       <div class="col-12 col-md-7 col-lg-5 p-3 p-md-3 p-lg-6" id="content">
-        <h2>2023 ZaiGeZaiGu Chinese New Year Gala Tickets On Sale Now</h2>
+        <h2>2023 ZGZG Summer Singing Contest
+        <p class="mt-1" style="font-size: 24px;">Open for registration!</p>
         <p>
-          <b>Date and Time: </b>Sat, January 21, 2023, 7PM – 10PM PST (New Year Festival 1-4PM) <br>
-          <b>Location: </b>San Jose Center for Performing Arts <br>
-          <b>Ticket Link: </b><a style="color: white;" href="https://zgzg.link/2023-ticket-website"><u>https://zgzg.link/2023-ticket-website</u></a><br>
-          <b>20%-off Promo code: </b> zgzg-early-bird (expires 01/08/23) <br>
+          Here, a spotlight shines just for you,<br>
+          A stage awaits your arrival,<br>
+          This will be a summer for you to achieve your musical dreams!<br><br>
+          Registration deadline: May 1st<br>
         </p>
-        <a href="https://zgzg.link/2023-ticket-website">
-          <button type="button" class="btn btn-light btn-primary btn-lg">Purchase Ticket</button>
-        </a>
+        <div style="display: flex; flex-direction: column; justify-content: flex-start; align-items: flex-start; gap: 10px">
+          <div style="display: flex; flex-wrap: wrap; justify-content: flex-start; align-items: flex-end; gap: 5px 10px;">
+            <a target="_blank" href="http://zgzg.link/2023s-signup">
```

Comment:
It links to a form in Chinese as well... Should we create separate forms in English here (I'm OK if it's intended)

Reply:
checked with Lizzi, it's ok to use the same link